test/unit/FormulateInputBox.test.js

```diff
@@ -207,4 +207,12 @@ describe('FormulateInputBox', () => {
     expect(wrapper.contains(FormulateInputGroup)).toBe(true)
     expect(wrapper.find('input[type="checkbox"]').exists()).toBe(false)
   })
+
+  it('renders multiple labels both with correct id', async () => {
+    const wrapper = mount(FormulateInput, { propsData: { type: 'checkbox', label: 'VueFormulate FTW!'} })
+    const id = wrapper.find('input[type="checkbox"]').attributes('id')
+    const labelIds = wrapper.findAll('label').wrappers.map(label => label.attributes('for'));
+    expect(labelIds.length).toBe(2);
```

Comment:
Not sure I understand what's going on here? Are the two labels representative of the SSR and client-side representations?

Reply:
There's no way to test the SSR issues using vue-test-utils, I would have to use another testing framework, so evetually we'll test this using Cypress — but in the meantime this test just makes sure that out of the box all the `<label for>`s match the main `<input id>`.  It's already a passing test, but worth having here in case of regression.